src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1418,7 +1415,7 @@ export default {
 			}
 		},
 		jumpToLoans() {
-			this.$refs.campaignLoanSection.scrollIntoView({ behavior: 'smooth' });
+			this.$refs.mlLoanDisplay[0].$refs.campaignLoanSection.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Is there guaranteed to have 1 item here in `mlLoanDisplay`? If not, optional chaining would be good.